src/Drupal/UserTrait.php

```
@@ -154,6 +154,64 @@ public function userSetLastLoginTime(string $name, string $datetime): void {
+   * @code
+   * Given the role "Content Manager" with the permissions "access content, create article content, edit any article content"
+   * @endcode
+   */
+  #[Given('the role :role_name with the permissions :permissions')]
+  public function userCreateRole(string $role_name, string $permissions): void {
+    $permissions = $this->helperSplitCommaSeparated($permissions);
+
+    $rid = strtolower($role_name);
+    $role_name = trim($role_name);
+
+    $existing_role = Role::load($rid);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Drupal\user\Entity\Role' in method 'userCreateRole'. (undefined)

(StaticAccess)